client/src/pages/DirectoryPage/DirectoryPeople.js

```diff
@@ -16,173 +16,324 @@ import image4 from "../../assets/images/profile-picture-4.png";
 import image5 from "../../assets/images/profile-picture-5.png";
 import Card from "../../components/Card/Card";
 
-function pillClick(val) {
-  console.log(val);
-}
+let friends = []
+let others = []
+let friendRequests = []
 
-const handleClick = e => {
-  ("/home");
-};
-
-// Page or
-const DirectoryPeople = props => {
-  return (
-    <div className="CommunityPage">
-      <NavBar page={0} profileImage={people} />
-      <Banner background={Banner__Directory}>
-        <h3 style={{ margin: 0 }}>Directory: My Friends</h3>
-        <InputTextField
-          type="text"
-          variation="wide"
-          name="search directory"
-          placeholder="Search Groups"
-        />
-      </Banner>
-      <div className="secondNav">
-        <a className="secondNav">
-          <Button text="My Friends" color="purple" size="long" handleClick={handleClick} />
-        </a>
-        <a className="secondNav" href="./groups">
-          <Button text="My Groups" color="outline" size="long" />
-        </a>
-      </div>{" "}
-      <div className="CommunityPage_SortByButton">
-        <div className="CommunityPage_SortByText">Sort By: Location </div>
-      </div>
-      <div className="CommunityPage_body">
-        <div>
-          <header className="CommunityPage__header">Location:</header>
+export default class DirectoryPeople extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      friends: [],
+      others: [],
+      friendRequests: []
+    };
+  }
+
+
+  componentDidMount() {
+    this.getFriends()
+    friends = this.state.friends
+  }
+
+
+  
+  
+  handleClick = e => {
```

Comment:
Your formatting here is really weird. Can you make sure you have prettier set up with VS Code with auto format on paste and on save?